agent/consul/state/catalog_test.go

```diff
@@ -2166,6 +2166,101 @@ func TestStateStore_DeleteCheck(t *testing.T) {
 	}
 }
 
+func ensureServiceVersion(t *testing.T, s *Store, ws memdb.WatchSet, serviceID string, expectedIdx uint64, expectedSize int) {
```

Comment:
We depend on go 1.9+ so you can use `t.Helper()` here which makes the error reporting much nicer.

Reply:
DONE